### PR TITLE
portable server using python3.8

### PR DESCRIPTION
**Pull request by tst-basilhuffman**

### CODE REVIEW

File: earth_enterprise/src/fusion/portableglobe/cutter/cgi-bin/common/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2.7
+#!/usr/bin/env python3
```
Comment (tst-dsugar) [inline earth_enterprise/src/fusion/portableglobe/cutter/cgi-bin/common/__init__.py:1]: These should all be python3.8, not python3

Comment (tst-basilhuffman) [reply earth_enterprise/src/fusion/portableglobe/cutter/cgi-bin/common/__init__.py]: done

**Review by tst-dsugar — APPROVED** [review]

**Review by tst-cjeffries — APPROVED** [review]
lgtm

**Review by tst-cjeffries — COMMENTED** [review]
lgtm

**Pull request merged**